code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -290,3 +290,7 @@
 	display_name = "shoulder sleeve patch"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/whalebone
```

Comment:
Need a to set this entry flags to GEAR_HAS_DESC_SELECTION

Reply:
Added to this and the whalebone.